tasklist/client/e2e/docs/docs-snapshots.spec.ts

```diff
@@ -0,0 +1,521 @@
+      {...alteredTaskList[1], assignee: 'demo'},
+      [
+        {
+          name: 'passengerName',
+          value: '"ARRON A. ARRONSON"',
+          isValueTruncated: false,
+          previewValue: '""',
+          draft: null,
+        },
+      ],
+      registerPassengerForm,
+    );
+
+    await taskDetailsPage.goto('1');
+
+    await expect(page.getByTestId('tasks-skeleton')).not.toBeVisible();
+
+    await expect(page.getByText('Register Passenger')).toBeVisible();
+    await expect(page.getByText('Name: ARRON A. ARRONSON')).toBeVisible();
+
+    await expect(page.getByText('Complete Task')).toBeVisible();
+    await expect(page.getByText('Complete Task')).toBeEnabled();
+
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('completed a task', async ({page, taskPanelPage}) => {
+    const completedTaskList = [
+      {
+        ...MOCK_TASKLIST[1],
+        assignee: 'demo',
+        taskState: 'COMPLETE',
+        completionDate: now,
+      },
+    ];
+    await mockTaskSearch(page, completedTaskList);
+    await mockTask(
+      page,
+      '1',
+      {...completedTaskList[0], assignee: 'demo'},
+      [
+        {
+          name: 'passengerName',
+          value: '"ARRON A. ARRONSON"',
+          isValueTruncated: false,
+          previewValue: '""',
+          draft: null,
+        },
+      ],
+      registerPassengerForm,
+    );
+
+    await taskPanelPage.goto({filter: 'completed', sortBy: 'completion'});
+
+    await expect(page.getByTestId('tasks-skeleton')).not.toBeVisible();
+
+    await taskPanelPage.openTask(completedTaskList[0].name);
+
+    await expect(page.getByText('Register Passenger')).toBeVisible();
+    await expect(page.getByText('Name: ARRON A. ARRONSON')).toBeVisible();
+
+    await expect(page.getByText('Complete Task')).toBeVisible();
+    await expect(page.getByText('Complete Task')).not.toBeEnabled();
+    await page.getByTestId('scrollable-list').click({position: {x: 0, y: 300}}); // Deselect the button
+
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('processes list', async ({page, processesPage}) => {
+    await mockProcessSearch(page, MOCK_PROCESSES);
+
+    await processesPage.goto();
+
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('processes search', async ({page, processesPage}) => {
+    await mockProcessSearch(page, MOCK_PROCESSES);
+
+    await processesPage.goto();
+    await processesPage.searchForProcess('order');
+    await expect(async () => {
+      expect(await page.getByTestId('process-tile').all()).toHaveLength(1);
+    }).toPass();
+
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('processes start form', async ({page, processesPage}) => {
+    await mockProcessSearch(page, MOCK_PROCESSES);
+    await mockForm(page, registerPassengerStartForm);
+
+    await processesPage.goto();
+    await page
+      .getByTestId('process-tile')
+      .filter({has: page.getByText('Register')})
+      .getByRole('button')
+      .click();
```

Comment:
I think you can just replace it with this, no?
Using the button name would also help to keep the tests working if we ever add another button in the tile

```suggestion
    await page
      .getByRole('button', {name: /start process/})
      .nth(0)
      .click();
```